third_party/docfx/templates/devsite/ManagedReference.html.primary.tmpl

```diff
@@ -1,7 +1,8 @@
 {{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
 {{!master(layout/_master.tmpl)}}
 
-<h1 class="page-title">{{#title}}{{title}}{{/title}}{{^title}}{{>partials/title}}{{/title}}</h1>
+<h1 class="page-title">{{#title}}{{title}}{{/title}}

```

Comment:
I think it's now safe to just write 
```
<h1 class="page-title">{{title}}</h1>
``` 
instead of 
```
<h1 class="page-title">{{#title}}{{title}}{{/title}}</h1>
```

Reply:
Done.